frontend/public/components/monitoring/metrics.tsx

```diff
@@ -10,10 +10,15 @@ import {
 import {
   ActionGroup,
   Button,
+  Dropdown as PFDropdown,
```

Comment:
Import as `PFDropdown` because we are also using the `Dropdown` component from `utils/` on this page, but the aim will be to refactor away that usage of the utils `Dropdown` with a future PR.